### PR TITLE
frontend/bugfix for IPA release

### PR DESCRIPTION
**Pull request by AmritpalSingh6749**
## Type of Change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description
Bugfix for IPA release ID_72, ID_73

### CODE REVIEW

File: Frontend/ui-customer/src/Strings/BN.js [hunk 1]
```diff
@@ -427,7 +427,7 @@ const bengaliStrings = {
   "NO_DONT" : "না, না",
   "YES_CANCEL_SEARCH" : "হ্যাঁ, অনুসন্ধান বাতিল করুন",
   "TRY_LOOKING_FOR_RIDES_AGAIN" : "এটি খুব ব্যস্ত দিন বলে মনে হচ্ছে। আপনি আবার রাইড খুঁজছেন চেষ্টা করতে পারেন",
-  "NO_TIP" : "0 টিপ্",
+  "NO_TIP" : "শূন্য টিপ",
```
Comment (MercyQueen) [inline Frontend/ui-customer/src/Strings/BN.js:430]: @AmritpalSingh6749 can we revert this change and add rupee symbol in front of 0 as this change was discussed with  sreedeep.

**Comment by MercyQueen** [comment]
can you add link for the issue IDs.

**Review by ashishsingh101 — APPROVED** [review]

File: Frontend/ui-customer/src/Screens/CustomerUtils/EmergencyContactsScreen/View.purs [hunk 2]
```diff
@@ -177,12 +177,11 @@ contactListView listItemm push state =
...
 showEmergencyContact listitemm push config =
   swipeRefreshLayout
-    [ width MATCH_PARENT
+    ([ width MATCH_PARENT
     , height MATCH_PARENT
     , background Color.blue600
     , weight 1.0
-    , id "2000040"
-    ]
+    ] <> if os == "IOS" then [] else [id "2000040"] )
```
Comment (kavya-shree-s) [inline Frontend/ui-customer/src/Screens/CustomerUtils/EmergencyContactsScreen/View.purs:184]: Use getNewIDWithTag

**Review by kavya-shree-s — DISMISSED** [review]
